Makefile

```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
 	@yarn run test-ci
 	@echo ""
 
-test-python:
-	@echo "--> Running Python tests"
-	# This gets called by getsentry
-	pytest tests/integration tests/sentry
```

Comment:
Why is this target being removed?

Reply:
It isn't used elsewhere after https://github.com/getsentry/getsentry/pull/7562. Also people are most likely not running that many tests at once in local dev.